### PR TITLE
Add toggle fullscreen support

### PR DESCRIPTION
**Pull request by coderkalyan**
This adds fullscreen support to views. When the keybinding Mod+shift+f is pressed, the current view will become fullscreen (if not already fullscreen), otherwise it will return to regular size.

### CODE REVIEW

File: src/output.zig [hunk 1]
```diff
@@ -133,7 +135,30 @@ pub const Output = struct {
     }
 
     pub fn arrange(self: *Self) void {
-        self.arrangeViews();
+        if (self.fullscreen_view) |view| {
+            if (self.fullscreen_view == null) {
+                Log.Debug.log("Yeah it is", .{});
+            }
```
Comment (ifreund) [inline src/output.zig:141]: Looks like leftover printf debugging.

**Comment by ifreund** [comment]
One last comment, I think this is actually closer to a monocle layout than true, exclusive full screen (which we probably want as well eventually). Could you adjust the nomenclature to respect that?

And thanks for working on this!

**Comment by coderkalyan** [comment]
Sure thing, I'll implement the changes tomorrow!

**Comment by ifreund** [comment]
I'm going to go ahead and close this as stalled. If you'd like to continue work on this feel free to resolve the conflicts and reopen or make a new PR.

**Pull request closed without merging**